driver-core/src/main/com/mongodb/connection/GenericWriteProtocol.java

```diff
@@ -29,6 +31,23 @@ public GenericWriteProtocol(final MongoNamespace namespace, final RequestMessage
     }
 
     @Override
+    protected void appendToWriteCommandResponseDocument(final RequestMessage curMessage, final RequestMessage nextMessage,
+                                                        final WriteConcernResult writeConcernResult, final BsonDocument response) {
+        throw new UnsupportedOperationException("Not implemented yet!");
```

Comment:
GenericWriteProtocol is only used for async, which is not yet generating events, so I think this is appropriate.